hook-tests/600-no-debian.test

```diff
@@ -24,14 +24,13 @@ else
   exit 1
 fi
 
-#third check the Packages section of dpkg.yaml
```

Comment:
Why is the comment removed?

Reply:
That's added back in now with the latest commit.